runner/docker/Dockerfile.live-base-comfyui

```diff
@@ -8,6 +8,7 @@ FROM ${BASE_IMAGE}
 ENV DEBIAN_FRONTEND=noninteractive
 ENV NVIDIA_VISIBLE_DEVICES=all
 ENV NVIDIA_DRIVER_CAPABILITIES=compute,utility,video
+ENV PATH="/workspace/miniconda3/bin:${PATH}"
```

Comment:
Some of the changes in this PR caused `dl_checkpoints.sh` to [fail](https://github.com/livepeer/ai-runner/actions/runs/14355264860/job/40243134959):

```
ModuleNotFoundError: No module named 'yaml'
...
ModuleNotFoundError: No module named 'psutil'
```

Reply:
This change was made early to cleanup environment activation, but it should be reverted back

Reply:
Another reason I removed it is there isn't a python env at `/workspace/ComfyUI` so this line didn't seem to make any sense and shouldn't have directly cause the issue. More likely it was the the change below on ln 80. https://github.com/livepeer/ai-runner/pull/479/files/a90f5b79cc5738021db37a4251c68e1029e170b1#diff-443cdad78f488f139f012984eac0c36f930e5ceb4ce1dd436f19c865e7ddc639L80. This change was made because it caused issues and seemed like a "hacky" way to activate the env, occasionally it would activate the env twice when working in a dev container and entirely prevented deactivation.

Model compilation should happen inside of the comfystream conda env so it's using the same packages as the runtime env